src/style.css

```diff
@@ -0,0 +1,49 @@
+html {
+  --star_acive: url(images/star-active.svg);
+  --star: url(images/star.svg);
+}
```

Comment:
You don't actually need to specify them as variables, you can use them directly, but it's ok/